airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -394,3 +394,8 @@
   dockerRepository: airbyte/source-typeform
   dockerImageTag: 0.1.0
   documentationUrl: https://docs.airbyte.io/integrations/sources/typeform
+- sourceDefinitionId: bfd1ddf8-ae8a-4620-b1d7-55597d2ba08c
+  name: BigQuery
+  dockerRepository: airbyte/source-bigquery
+  dockerImageTag: 0.1.0
+  documentationUrl: https://docs.airbyte.io/integrations/sources/bigquery
```

Comment:
Is this pr blocked until the docs are added here?

Reply:
Also, this is the "correct" link once it's created but `airbyte-integrations/connectors/source-bigquery/src/main/resources/spec.json` points to `/source/bigquery` instead.

Reply:
yes, @heade is working on finishing docs